scheduler/system_sched.go

```diff
@@ -297,13 +297,28 @@ func (s *SystemScheduler) computePlacements(place []allocTuple) error {
 					desired := s.plan.Annotations.DesiredTGUpdates[missing.TaskGroup.Name]
 					desired.Place -= 1
 				}
+				continue
```

Comment:
Would be good to add a comment on why we continue here

Reply:
@preetapan I put the comment up at the top of the `option == nil` where it applies to all 3 of the `continue` statements

Reply:
probably worth commenting why why we don't increment failedTGAllocs if `NodeFiltered > 0`.

Reply:
yeah that ^ is what i was getting at - documenting that we don't increment failedTGAllocs because  nodeFiltered is > 0 only when there's a constraint that didn't match, and a failed constraint condition should not be counted as a failure to place that task group.

Reply:
Let me know if the additional comments provide enough context.

Reply:
lgtm :shipit: